NachoSpigot-Server/src/main/java/net/minecraft/server/TileEntitySign.java

```diff
@@ -121,7 +122,10 @@ public boolean b() {
     }
 
     public void a(EntityHuman entityhuman) {
-        this.h = entityhuman;
+        // Paper start
+        //this.h = entityhuman;
+        signEditor = entityhuman != null ? entityhuman.getUniqueID() : null;
+        // Paper end
     }
 
     public EntityHuman c() {
```

Comment:
When possible, remove this method to fix the compile error (can't add suggestions to non-edited code)